microservices-swift/README.rtf

```diff
@@ -0,0 +1,184 @@
+{\rtf1\ansi\deff0{\fonttbl{\f0 \fswiss Helvetica;}{\f1 Courier;}}
```

Comment:
Trash this as well

Reply:
f83a03b414306be44258998ad446b86d505d0cfd